src/websocketClient.ts

```diff
@@ -1,9 +1,11 @@
 import { pipe, prop } from "ramda";
-import * as WebSocket from "ws";
+import * as ReconnectingWS from "reconnecting-websocket";
```

Comment:
Is there a reason for using both the import all and the default import for the `reconnecting-websocket` package?

Reply:
Yeah... it's kind of a typing problem/export problem.
You're SUPPOSED to be able to do `import ReconnectingWebsocket from "reconnecting-websocket"`, but that doesn't actually work in execution - so I was able to trick it this way and still get the types working.
I added tests to cover this case.

Reply:
Eww, how lame. Okay, got it 👍